schema/types/Answer.js

```diff
@@ -0,0 +1,40 @@
+const { GraphQLObjectType, GraphQLString, GraphQLBoolean, GraphQLInt } = require("graphql");
+
+module.exports = new GraphQLObjectType({
+  name : "Answer",
+  description : "An answer",
+
+  fields : () => ({
+    id : {
+      type: GraphQLInt
```

Comment:
If it causes problems with implicit casting then don't worry about it. It was only a small point and int makes sense given the underlying types of the db columns.